ci-operator/step-registry/single-node/e2e/test/single-node-e2e-test-commands.sh

```diff
@@ -152,6 +152,14 @@ ibmcloud)
     IC_API_KEY="$(< "${CLUSTER_PROFILE_DIR}/ibmcloud-api-key")"
     export IC_API_KEY
     ;;
+powervs*)
```

Comment:
The only difference between those two files should be `--max-parallel-tests 15 \`

Reply:
https://github.com/openshift/release/pull/43704

Reply:
Ideally the max parallel tests should be configurable on the original file but I got pushback when asking to make it configurable

Reply:
#43704 merged, so please add this to the original file as well

Reply:
@omertuc 
I just rebased and noticed that my small change to this file remains the same.  It will cleanly apply to your new changes.  Does this satisfy your concern?

Reply:
Or do you want me to apply my changes "upstream" to that other file?

Reply:
>Or do you want me to apply my changes "upstream" to that other file?

This

Reply:
In that case, wouldn't I have to change both files then?

Reply:
Yes

Reply:
Done!